app/views/content/initial-teacher-training.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Initial teacher training (ITT)
+related_content:
+    Diary of a trainee teacher : "/blog/a-diary-of-a-trainee-teacher"
+    Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider"
+    Train to be a teacher : "/train-to-be-a-teacher"
```

Comment:
The link test will fail here until the #2474 has been merged as the page doesn't yet exist.

If this is planned to go out at the same time as that page we can merge this branch into that one, or comment it out for the time being.

Reply:
@peteryates how do we merge this branch with the main train to be a teacher one? :)

Reply:
If you click 'Edit' at the top of this PR you can change the target branch to 'new-nav'.